python/pyspark/pandas/data_type_ops/null_ops.py

```diff
@@ -43,6 +43,22 @@ class NullOps(DataTypeOps):
     def pretty_name(self) -> str:
         return "nulls"
 
+    def eq(self, left: IndexOpsLike, right: Any) -> SeriesOrIndex:
+        _sanitize_list_like(right)
+        result = pyspark_column_op("__eq__")(left, right)
+        if is_remote():
+            # TODO(SPARK-43877): Fix behavior difference for compare binary functions.
+            result = result.fillna(False)
+        return result
+
+    def ne(self, left: IndexOpsLike, right: Any) -> SeriesOrIndex:
+        _sanitize_list_like(right)
+        result = pyspark_column_op("__ne__")(left, right)
+        if is_remote():
```

Comment:
Can you put this into `pyspark_column_op`, and add a parameter to deduplicate the codes? e.g., `fillna=value`

Reply:
Sounds good. Let me address